bgpd/bgp_conditional_adv.c

```diff
@@ -269,11 +285,14 @@ static int bgp_conditional_adv_timer(struct thread *t)
 						__func__, peer->host,
 						get_afi_safi_str(afi, safi,
 								 false));
-
-				paf = peer_af_find(peer, afi, safi);
 				if (paf) {
 					update_subgroup_split_peer(paf, NULL);
 					subgrp = paf->subgroup;
+					SUBGRP_PEER(PAF_SUBGRP(paf))
```

Comment:
dumb question: Isn't this update_type set above? I mean _double_ set here.

Reply:
Hm, so it is.